### PR TITLE
Update version to 2021.2

### PR DESCRIPTION
**Pull request by homksei**

### CODE REVIEW

File: makefile.ver [hunk 1]
```diff
@@ -15,10 +15,10 @@
 #===============================================================================
 
 MAJOR   =       2021
-MINOR   =       1
-UPDATE  =       9
+MINOR   =       0
```
Comment (napetrov) [inline makefile.ver:18]: should be 2

File: makefile.ver [hunk 2]
```diff
@@ -15,10 +15,10 @@
...
+UPDATE  =       2
```
Comment (napetrov) [inline makefile.ver:19]: should be 0

**Review by maria-Petrova — APPROVED** [review]

**Pull request merged**